### ISSUE TITLE
First setup of the app

### ISSUE THREAD

**Issue by ernesst** [strukturag/nextcloud-spreedme#49]
I would like to use this app as a personal use with persons not familiar with computer. 

When they click on an invite link, the first thing appearing are the parameter box. On a phone people doesn't recognize as a tchat and get lost; i guess because the setting area takes the overall screen.  

Would it be possible to not bring this box by default for the first connection ?

Rgds

**Comment by leonklingele** [comment]
> When they click on an invite link, the first thing appearing are the parameter box.

Are you talking about the settings panel (on the right)?

**Comment by ernesst** [comment]
Yes correct the setting panel on the right at the first connexion.
